neptune/new/exceptions.py

```diff
@@ -275,9 +275,11 @@ def __init__(self, run_id: str) -> None:
         super().__init__("Run {} not found.".format(run_id))
 
 
-class RunUUIDNotFound(NeptuneException):
-    def __init__(self, run_id: str):
-        super().__init__("Run with ID {} not found. Could be deleted.".format(run_id))
+class ContainerUUIDNotFound(NeptuneException):
+    def __init__(self, container_id: str):
+        super().__init__(
+            "Container with ID {} not found. Could be deleted.".format(container_id)
```

Comment:
I think this should get `container_type` to allow for nicer user-facing errors. Cf. spec in NPT-11119